core/domain/services/registration/form/v1/RegForm.php

```diff
@@ -0,0 +1,292 @@
+<?php
+
+namespace EventEspresso\core\domain\services\registration\form\v1;
+
+use DomainException;
+use EE_Checkbox_Multi_Input;
+use EE_Error;
+use EE_Form_Input_Base;
+use EE_Form_Section_HTML;
+use EE_Form_Section_Proper;
+use EE_Line_Item_Display;
+use EE_Question;
+use EE_Registration;
+use EE_Registration_Config;
+use EE_SPCO_Reg_Step_Attendee_Information;
+use EE_Template_Layout;
+use EEH_Autoloader;
+use EEH_Line_Item;
+use EventEspresso\core\exceptions\EntityNotFoundException;
+use EventEspresso\core\exceptions\InvalidDataTypeException;
+use EventEspresso\core\exceptions\InvalidInterfaceException;
+use EventEspresso\core\services\loaders\LoaderFactory;
+use InvalidArgumentException;
+use ReflectionException;
+
+/**
+ * Class RegForm
+ *
+ * @author  Brent Christensen
+ * @package EventEspresso\core\domain\services\registration\form
+ * @since   $VID:$
+ */
+class RegForm extends EE_Form_Section_Proper
+{
+
+    /**
+     * @var bool
+     */
+    private $print_copy_info = false;
+
+    /**
+     * @var EE_Registration_Config
+     */
+    public $reg_config;
+
+    /**
+     * @var int
+     */
+    protected $reg_form_count = 0;
+
+    /**
+     * @var EE_SPCO_Reg_Step_Attendee_Information
+     */
+    public $reg_step;
+
+    /**
+     * @var array
+     */
+    private $template_args = [];
+
+
+    /**
+     * RegForm constructor.
+     *
+     * @param EE_SPCO_Reg_Step_Attendee_Information $reg_step
+     * @param EE_Registration_Config                $reg_config
+     * @throws ReflectionException
+     * @throws EE_Error
+     */
+    public function __construct(
+        EE_SPCO_Reg_Step_Attendee_Information $reg_step,
+        EE_Registration_Config $reg_config
+    ) {
+        $this->reg_step                   = $reg_step;
+        $this->reg_config                 = $reg_config;
```

Comment:
my autoformat gets a little overzealous, i can manually clean this up

Reply:
ahh... this was leftover from a previous autoformat that was lining things up with some other code that has since been removed. autoformatting again, fixed this :)